src/ucp/tag/rndv.c

```diff
@@ -283,7 +367,9 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_get_zcopy, (self),
         length = ucp_mtu - ((uintptr_t)rndv_req->send.buffer % align);
     } else {
         length = ucs_min(rndv_req->send.length - offset,
-                         ucp_ep_config(rndv_req->send.ep)->tag.rndv.max_get_zcopy);
+                         ucs_min(ucp_ep_config(rndv_req->send.ep)->tag.rndv.max_get_zcopy,
+                                 (rndv_req->send.length /
```

Comment:
calculation is not correct, e.g rndv_req->send.length=100, ucp_ep_rndv_num_lanes(rndv_req->send.ep)=2,
it will be 51+49 instead of 50+50
see https://stackoverflow.com/a/2745086